.github/workflows/semver-checks.yml

```diff
@@ -4,8 +4,20 @@ on:   # yamllint disable-line rule:truthy
 name: Check semver breaks
 
 jobs:
+  Prepare:
+    runs-on: ubuntu-latest
+    outputs:
+      stable_version: ${{ steps.read_toolchain.outputs.stable_version }}
+    steps:
+      - name: "Checkout repo"
+        uses: actions/checkout@v4
+      - name: "Read stable version"
+        id: read_toolchain
+        run: echo "stable_version=$(cat ./.github/workflows/stable-version)" >> $GITHUB_OUTPUT
+
```

Comment:
This entire job is not needed since we're using `cat` directly.

Reply:
Good catch! Thanks

Reply:
Why is it still here?

Reply:
I've removed the `rust_stable_version`, that's why I told you "Good catch".
But this is needed for https://github.com/rust-bitcoin/rust-bitcoin/blob/3a19dece44c82f98a17d23bab5f373130bea6a30/.github/workflows/semver-checks.yml#L38-L39 and  https://github.com/rust-bitcoin/rust-bitcoin/blob/3a19dece44c82f98a17d23bab5f373130bea6a30/.github/workflows/semver-checks.yml#L78-L79

Reply:
Could you please clean up the history to not add it in 2e0f15a47e1a62d4405d82fb717f8e80fa15a4e8 so it won't need to be removed later?

Reply:
Done, removed from 4edd504cb60397cfe143607ef5db8784598cbd06